mora/cli.py

```diff
@@ -540,3 +541,205 @@ def fixroots(**kwargs):
                                        unitid),
                         unit,
                     )
+
+    @app.cli.command('add-engagements')
+    @click.argument('sheet', type=click.Path())
+    @click.argument('target')
+    @click.option('--delimiter', '-d', default=';')
+    @click.option('--include', '-I', multiple=True,
+                  help='include only employees matching the given patter')
+    @click.option('--jobs', '-j', default=1, type=int,
+                  help='Amount of parallel requests.')
+    def engagements(sheet, target, delimiter, include, jobs):
```

Comment:
This should be renamed to better reflect what it actually does. I'm guessing this is the general-ish CLI command for importing users from spreadsheets?